release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -89,6 +89,12 @@ For `console` and `downloads` routes, custom routes functionally is now implemen
 [id="ocp-4-8-networking"]
 === Networking
 
+[id="ocp-4-8-network-policy-host-network-egress-router-pods"]
+==== Network policy supports selecting host network egress router pods
```

Comment:
This is not about *egress* routers, it's about ingress routers. (Except AFAIK we never call them "ingress routers", they're just "routers".)

Reply:
@danwinship wow not sure why I thought this was egress at all. That's a different epic. Thanks!